examples/ecmascript/es5.ohm

```diff
@@ -311,20 +308,12 @@ ES5 {
                  | MemberExpression Arguments         -- memberExpExp
 
   // @returns an array of ASTs
-  Arguments = "(" ")"                -- empty
-            | "(" ArgumentList ")"   -- nonEmpty
-
-  // @returns an array of ASTs
-  ArgumentList = AssignmentExpressionComma* AssignmentExpression -- many
-               | AssignmentExpression                            -- one
-
-  AssignmentExpressionComma = AssignmentExpression ","
+  Arguments = "(" ListOf<AssignmentExpression, ","> ")"
 
   LeftHandSideExpression = CallExpression
                          | NewExpression
 
-  PostfixExpression = LeftHandSideExpression #(spacesNoNL "++")  -- postIncrement
-                    | LeftHandSideExpression #(spacesNoNL "--")  -- postDecrement
+  PostfixExpression = LeftHandSideExpression #(spacesNoNL ("++"|"--"))  -- post
```

Comment:
I think it's also better to keep these cases separate, for convenience in writing semantic actions.